.meta/sinks/sematext_metrics.toml.erb

```diff
@@ -0,0 +1,64 @@
+[sinks.sematext_metrics]
+title = "Sematext Metrics"
+beta = true
+common = false
+delivery_guarantee = "at_least_once"
+description = """\
+[Sematext][urls.sematext] is a hosted monitoring platform for metrics based on \
+InfluxDB. Providing powerful monitoring and management solutions to \
+monitor and observe your apps in real-time.\
+"""
+egress_method = "batching"
+features = [
+  "Send metrics to the Sematext monitoring service.",
+  "Batch data to maximize throughput.",
+  "Automatically retry failed requests, with backoff.",
+  "Buffer your data in-memory or on-disk for performance and durability.",
+]
+function_category = "transmit"
+healthcheck = true
+input_types = ["metric"]
+requirements = {}
+service_providers = ["Sematext"]
+write_to_description = "[Sematext][urls.sematext] via the [InfluxDB API v1][urls.influxdb_http_api_v1]."
+
+<%= render("_partials/fields/_component_options.toml", type: "sink", name: "sematext_metrics") %>
+
+<%= render("_partials/fields/_batch_options.toml", namespace: "sinks.sematext_metrics.options", common: false, max_bytes: 10490000, max_events: nil, timeout_secs: 1) %>
+
+<%= render(
+  "_partials/fields/_buffer_options.toml",
+  namespace: "sinks.sematext_metrics.options",
+  common: false
+) %>
+
+<%= render(
+  "_partials/fields/_request_options.toml",
+  namespace: "sinks.sematext_metrics.options",
+  common: false,
+  rate_limit_duration_secs: 1,
+  rate_limit_num: 5,
+  retry_initial_backoff_secs: 1,
+  retry_max_duration_secs: 10,
+  timeout_secs: 60
+) %>
+
+[sinks.sematext_metrics.options.region]
+type = "string"
+required = false
+examples = ["na", "eu"]
+description = "The region destination to send metrics to. This option is required if `host` is not set."
+
+[sinks.sematext_metrics.options.host]
+type = "string"
+required = false
+examples = ["http://127.0.0.1", "http://example.com"]
+description = "The host that will be used to send metrics to. This option is required if `region` is not set."
```

Comment:
@binarylogic Is there a way to flag these with `relevant_when` given that the condition is for when the other is unset?

Reply:
Yes:

```
relevant_when = {field = ""}
```

Will do the trick.